src/Component/spec/Generator/RandomnessGeneratorSpec.php

```diff
@@ -25,13 +23,41 @@ function it_implements_randomness_generator_interface(): void
         $this->shouldImplement(RandomnessGeneratorInterface::class);
     }
 
-    function it_implements_legacy_randomness_generator_interface(): void
+    function it_generates_random_uri_safe_string_of_length(): void
     {
-        $this->shouldImplement(LegacyRandomnessGeneratorInterface::class);
+        $length = 9;
+
+        $this->generateUriSafeString($length)->shouldBeString();
+        $this->generateUriSafeString($length)->shouldHaveLength($length);
+    }
+
+    function it_generates_random_numeric_string_of_length(): void
+    {
+        $length = 12;
+
+        $this->generateNumeric($length)->shouldBeString();
+        $this->generateNumeric($length)->shouldBeNumeric();
+        $this->generateNumeric($length)->shouldHaveLength($length);
+    }
+
+    function it_generates_random_int_in_range(): void
+    {
+        $min = 12;
+        $max = 2000000;
+
+        $this->generateInt($min, $max)->shouldBeInt();
+        $this->generateInt($min, $max)->shouldBeInRange($min, $max);
     }
 
-    function it_should_be_an_alias_of_randomness_generator(): void
+    public function getMatchers(): array
     {
-        $this->shouldBeAnInstanceOf(NewRandomnessGenerator::class);
+        return [
+            'haveLength' => function ($subject, $length) {
```

Comment:
Is it applicable over here?
```suggestion
            'haveLength' => function (string $subject, int $length): bool {
```